bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -0,0 +1,208 @@
+#!/bin/env python
+""" Apply a naive mass binning, assuming that each bin is fully independent, which 
+is a conservative estimate. This clusters to find the most significant foreground, but
+leaves the background triggers alone. """
+
+import h5py, numpy, argparse, logging, pycbc, pycbc.events, lal
+import pycbc.version
+from scipy.special import comb
+import matplotlib as mpl
+mpl.use('agg')
+import matplotlib.pyplot as plt
+
+def com(f, files, group):
+    """ Combine the same column from multiple files and save to a third"""
+    f[group] = numpy.concatenate([fi[group][:] if group in fi else \
+                                   numpy.array([], dtype=numpy.uint32) for fi in files])
+
+def all_keys_set(files):
+    key_set = []
+    for fi in files:
+        for k in fi.keys():
+            if isinstance(fi[k], h5py.Dataset):
+                key_set.append(k)
+            else:
+                key_set += get_all_subkeys(fi,k)
+    return set(key_set)
+
+def get_all_subkeys(fi,k):
+    subkey_set = []
```

Comment:
not clear why this is called 'set' as it's a list .. ?

Reply:
This was to make it clear that it is being returned as a set (eventually in the case of the subkeys) - but this can be changed